manifests/spel-minimal-rhel-7-hvm.manifest.txt

```diff
@@ -1,20 +1,20 @@
-Red Hat Enterprise Linux Server release 7.7 (Maipo)
-aws-cli/1.18.30 Python/2.7.5 Linux/3.10.0-1062.4.3.el7.x86_64 botocore/1.15.30
-acl-2.2.51-14.el7.x86_64
+Red Hat Enterprise Linux Server release 7.8 (Maipo)
```

Comment:
Need to note in the changelog that RHEL is now 7.8...

Reply:
Fixed it